src/factories/workflow-specific/PIM_WorkflowFactory_v1.sol

```diff
@@ -0,0 +1,228 @@
+        address _owner,
+        address _trustedForwarder
+    ) Ownable(_owner) ERC2771Context(_trustedForwarder) {
+        orchestratorFactory = _orchestratorFactory;
+    }
+
+    //--------------------------------------------------------------------------
+    // Public Mutating Functions
+
+    /// @inheritdoc IPIM_WorkflowFactory_v1
+    function createPIMWorkflow(
+        IOrchestratorFactory_v1.WorkflowConfig memory workflowConfig,
+        IOrchestratorFactory_v1.ModuleConfig memory authorizerConfig,
+        IOrchestratorFactory_v1.ModuleConfig memory paymentProcessorConfig,
+        IOrchestratorFactory_v1.ModuleConfig[] memory moduleConfigs,
+        IPIM_WorkflowFactory_v1.PIMConfig memory PIMConfig
+    )
+        external
+        returns (IOrchestrator_v1 orchestrator, ERC20Issuance_v1 issuanceToken)
+    {
+        // deploy issuance token
+        ERC20Issuance_v1 issuanceToken = new ERC20Issuance_v1(
+            PIMConfig.issuanceTokenParams.name,
+            PIMConfig.issuanceTokenParams.symbol,
+            PIMConfig.issuanceTokenParams.decimals,
+            PIMConfig.issuanceTokenParams.maxSupply,
+            address(this) // assigns owner role to itself initially to manage minting rights temporarily
+        );
+
+        // mint initial issuance supply to recipient
+        issuanceToken.mint(
+            PIMConfig.recipient, PIMConfig.bcProperties.initialIssuanceSupply
+        );
+
+        // assemble fundingManager config and deploy orchestrator
+        IOrchestratorFactory_v1.ModuleConfig memory fundingManagerConfig =
+        IOrchestratorFactory_v1.ModuleConfig(
+            PIMConfig.metadata,
+            abi.encode(
+                address(issuanceToken),
+                PIMConfig.bcProperties,
+                PIMConfig.collateralToken
+            )
+        );
+        IOrchestrator_v1 orchestrator = IOrchestratorFactory_v1(
+            orchestratorFactory
+        ).createOrchestrator(
+            workflowConfig,
+            fundingManagerConfig,
+            authorizerConfig,
+            paymentProcessorConfig,
+            moduleConfigs
+        );
+
+        // get bonding curve (= funding manager) address
+        address fundingManager = address(orchestrator.fundingManager());
+        issuanceToken.setMinter(fundingManager, true);
+
+        // revoke minter role from factory
+        issuanceToken.setMinter(address(this), false);
+
+        // if renounced token flag is set, renounce ownership over token, else transfer ownership to initial admin
+        if (PIMConfig.isRenouncedIssuanceToken) {
+            _transferTokenOwnership(issuanceToken, address(0));
+        } else {
+            _transferTokenOwnership(
+                issuanceToken, PIMConfig.issuanceTokenParams.initialAdmin
+            );
+        }
+
+        // if renounced workflow flag is set, renounce admin rights over workflow, else transfer admin rights to initial admin
+        if (PIMConfig.isRenouncedWorkflow) {
```

Comment:
I think we'de need some kind of check here that if the deployer renounces workflow ownership, the project fees have to be set to 0. If not, we'd end up with stuck funds.  Alternatively we could do the thing we mentioned of giving admin role to the factory, and storing a mapping(deployer => worklow) that routes the withdrawProjectCollateralFee() function. (Although what happens with lost private keys in that case? maybe easier to just set fees to 0)

Reply:
We have decided in the last tech sync that we go with the approach where fees can be claimed via the factory. This is currently being implemented in a follow-up PR (draft: https://github.com/InverterNetwork/contracts/pull/631).

Lost keys is a "pech gehabt" scenario imo. This is the same for lost admin keys in a regular workflow, no?